src/libs/actions/IOU.js

```diff
@@ -178,6 +178,14 @@ function setMoneyRequestAmount_temporaryForRefactor(transactionID, amount, curre
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {amount, currency});
 }
 
+/**
+ * Reset the money request amount, discarding the user-provided value. In the case of distance requests, this will effectively re-enable the default behavior of automatic amount calculation.
+ * @param {String} transactionID
+ */
+function resetMoneyRequestAmount_temporaryForRefactor(transactionID) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {amount: CONST.IOU.DEFAULT_AMOUNT});
```

Comment:
I believe this lead to [a regression](https://github.com/Expensify/App/issues/33878) because it doesn't not take `isDraft` into account like this:

```
`${isDraft ? ONYXKEYS.COLLECTION.TRANSACTION : ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`
```

Reply:
@tgolen I will raise a quick PR to fix it

Reply:
I believe that [this](https://github.com/Expensify/App/issues/33885) may be another regression w/ the same cause